docs/processors/standalone/conduit-processor-template.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Conduit Processor Template"
+sidebar_position: 1
+---
+
+## Initializing a Conduit Processor Project
+
+To begin the development of a custom Conduit processor, it is recommended developers should initialize their project using the [Conduit processor template](https://github.com/ConduitIO/conduit-processor-template). This template streamlines the setup process by providing a fundamental project structure, along with utility configurations for GitHub actions and a Makefile.
+
+Included in the Conduit Processor Template are:
+
+- Base code for processor's config and lifecycle.
```

Comment:
Reeeeally a nitpick, but I think it would be good to be consistent with the usage of "config" and "configuration" with "configuration" being the preferred option?